pypeit/flatfield.py

```diff
@@ -1074,6 +1066,16 @@ def fit(self, spat_illum_only=False, debug=False):
             # TODO: Add some code here to treat the edges and places where fits
             #  go bad?
 
+            # Build wavelength image AFTER the slits have been tweaked.  This is
+            #  the only place in this method that uses it.  It is not always used, 
+            #  but for convenience done here
+            slitmask = self.slits.slit_img(initial=True, 
```

Comment:
I don't think you want `initial=True` here.  If the code is working as documented, this should force use of the initial traces, not the tweaked ones.  Remove this so that `initial` is False.

Reply:
For my particular test case, both `initial=True` and `initial=False` work identically.  I have made the requested change.

Reply:
I need to go back and triple-check my test case with various permutations of `initial` for `slits.slit_img()` and placement of the `build_waveimg` in this function.  I will mark this PR as draft for now while I resolve these questions.  Thanks for your patience.